sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -704,6 +707,7 @@ This functionality allows running multiple actions in one or more documents. Act
         docNumber = 1;
         foreach (AnalyzeSentimentActionResult analyzeSentimentActionsResult in analyzeSentimentResults)
         {
+            Console.WriteLine($" Action name: {analyzeSentimentActionsResult.ActionName}");
```

Comment:
did you intentionally not add this to the linked entities output? edit: same question for samples

Reply:
correct. I added a comment to the issue so I remember to add it here. https://github.com/Azure/azure-sdk-for-net/issues/21798

Reply:
oh right, forgot about that bug